src/main/java/visualization/ImageHandler.java

```diff
@@ -138,4 +138,35 @@ public static void saveImage(JPanel p) throws IOException {
         newFile.mkdirs();
         ImageIO.write(img, "png", newFile);
     }
-}
\ No newline at end of file
+
```

Comment:
Copy paste funkcji Kamila i Łukasza z tym, że zwraca stringa

Reply:
Myślę, że możesz im zmienić na zwracanie Stringa po prostu jak mają void, copy paste nigdy nie jest dobry. Można im dać znać albo oznaczyć jakoś(?). Bo tylko zwracany typ się zmienił z tego co widzę.

Reply:
To napiszę do nich